### PR TITLE
Update internal error documentation

### PR DESCRIPTION
**Pull request by grubernaut**

### CODE REVIEW

File: packer/builds/troubleshooting.html.md [hunk 1]
```diff
@@ -62,14 +62,16 @@ for versioning and automated builds.
 This is a short list of internal errors and what they mean.
 
 - SIC-001: Your data was being ingressed from GitHub but failed
-to properly unpack. This can be caused by bad permissions, improper
-symlinks or very large repository sizes. _**Note:** Most often this error occurs 
+to properly unpack. This can be caused by bad permissions, using
+symlinks or very large repository sizes. Using symlinks inside of the
+packer directory will result in this internal error.
```
Comment (pearkes) [inline packer/builds/troubleshooting.html.md]: What's the packer directory in this case?

Comment (grubernaut) [inline packer/builds/troubleshooting.html.md]: Either the root of the repository (default), or the user specified packer directory in the build configuration's integration settings.

Comment (pearkes) [inline packer/builds/troubleshooting.html.md]: K cool, I would clarify that, wasn't immediately obvious to me.

**Comment by grubernaut** [comment]
@pearkes updated to include wording

**Comment by pearkes** [comment]
LGTM

**Pull request merged**